client.go

```diff
@@ -177,3 +227,22 @@ func (c *IClient) SearchWithTag(tag string) (*SearchResponse, error) {
 
 	return c.parseSearchResponse(resp)
 }
+
+func (c *IClient) GenerateSignedProxyUrl(assetID, proxyID string) (string, error) {
```

Comment:
can you put a comment above here about how to figure out what the proxyID is? The idea was that we would have an assetID given, but now a proxyID is needed as well, how do you get that? do you always just use 0 for the default or something?